### DISCUSSION TITLE
Support for ASP.NET core application

### DISCUSSION THREAD

**Discussion by devagupt** [scalar/scalar#762] [Ideas]
Hi team,
It will be great if the scalar can be provided in the nuget form for ASP.NET core applications (something like [this](https://www.nuget.org/packages/swashbuckle.aspnetcore.swaggerui/#readme-body-tab))
We are looking for the alternative of SwaggerUI and scalar seems a good option. However, due to non availability for .NET limits us to use it. I will be happy to assist in any further queries.

**Comment by marclave** [comment]
hey @devagupt ! appreciate the comment, we will look into making a great integration for ASP.NET and update this discussion when we have something solidified planned ✨

**Comment by marclave** [comment]
@devagupt 👀 https://github.com/scalar/scalar/pull/1737

with .net version 9.0 theres OpenAPI support out of the box! which is so exciting ✨ can follow along with this PR here for our official integration ✨ 

we can still look back to previous versions and open the discussion there but an exciting future :)

**Comment by xC0dex** [comment]
I think this discussion can be safely closed as we now provide the `Scalar.AspNetCore` package 🥳.
